File: xFit/UI/xfit_mobile/lib/models/uposlenik.dart
import 'package:json_annotation/json_annotation.dart';

part 'uposlenik.g.dart';

@JsonSerializable()
class Uposlenik{
	int? uposlenikId;
  String? ime; 
  String? prezime;
  DateTime? datumRodjenja;
  int?korisnikId;
  
  Uposlenik(this.uposlenikId,this.ime,this.prezime,this.datumRodjenja,this.korisnikId);

  factory Uposlenik.fromJson(Map<String, dynamic> json) => _$UposlenikFromJson(json);

  Map<String, dynamic> toJson() => _$UposlenikToJson(this);

}

File: xFit_rabbit/xFit_rabbit/Data/Uposlenik.cs
using System;
using System.Collections.Generic;

namespace xFit_rabbit.Data;

public partial class Uposlenik
{
    public int UposlenikID { get; set; }

    public string? Ime { get; set; }

    public string? Prezime { get; set; }

    public DateTime? DatumRodjenja { get; set; }

    public int? KorisnikID { get; set; }

    public virtual Korisnik? Korisnik { get; set; }

    public virtual ICollection<Rezervacija> Rezervacijas { get; } = new List<Rezervacija>();
}
